src/processes.c

```diff
@@ -2023,12 +2023,15 @@ static int ps_read (void)
 /* #endif HAVE_THREAD_INFO */
 
 #elif KERNEL_LINUX
-	int running  = 0;
-	int sleeping = 0;
-	int zombies  = 0;
-	int stopped  = 0;
-	int paging   = 0;
-	int blocked  = 0;
+	int running               = 0;
+	int sleeping              = 0;
+	int zombies               = 0;
+	int stopped               = 0;
+	int paging                = 0;
+	int blocked               = 0;
+	/* returns the number of processes with a command line */
+	int cmdline_processes     = 0;
```

Comment:
One space before the `=` is enough here. You can align the rest of the `=` signs with this one.
Also everywhere below.